authn-authz/sca-scanning-results.md

```diff
@@ -0,0 +1,14 @@
+# SCA scanning results
+## 1.0.1-beta.1
+
+### OSSPI Binary Scan
+
+* Date: March 28, 2022
+* Results: No known vulnerabilities. See scan results in [OSSPI](https://osspi.eng.vmware.com/audit-reports/24019/oss-packages/all?limit=25&offset=0&audit_id=24019&ordering=id).
```

Comment:
```suggestion
* Results: No known vulnerabilities.
```

Readers won't be able to access eng.vmware.com addresses

Reply:
okay removed the OSSPI link